### ISSUE TITLE
ModelReplacement Compatability

### ISSUE THREAD

**Issue by WanderingRei** [Zaggy1024/LC_OpenBodyCams#7]
### I believed this mod is not compatible with ModelReplacement?
![image](https://github.com/Zaggy1024/LC_OpenBodyCams/assets/130099855/1ee41825-b6ed-4fca-b5ce-ad62d5825aea)

**Comment by Zaggy1024** [comment]
Yes, it isn't yet compatible, but that's definitely something on the roadmap.
